content/api/commands/intercept.md

```diff
@@ -120,6 +145,12 @@ See [Request/Response Modification with `routeHandler`](#Request-Response-Modifi
 
 ## Examples
 
+<Alert type="info">
+
+`cy.intercept` can be used solely for spying: to passively listen for matching routes and apply [aliases](#Aliasing-a-Route) to them without manipulating the request or its response in any way. This alone is powerful as it allows you to [wait](#Waiting-on-a-request) for these requests, resulting in more reliable tests.
+
+</Alert>
+
 ### Matching `url`
```

Comment:
I'm looking at the "Matching url" and "Matching method" examples, and it's unclear what they actually _do._ Do I also need to alias them with `.as()` for them to be useful?

Reply:
Let's revisit this in the next iteration.